seml/config.py

```diff
@@ -268,6 +269,17 @@ def read_config(config_path):
         logging.error("Please specify a 'seml' dictionary in the experiment configuration.")
         sys.exit(1)
     seml_dict = config_dict['seml']
+    config_dir = os.path.abspath(os.path.dirname(config_path))
+    seml_dict['config_dir'] = config_dir
+    os.chdir(config_dir)
```

Comment:
We could call this `cwd` or `working_dir` instead and have it contain the root_dir or config_dir, depending on what is valid.

Reply:
We just need to keep backwards compatibility in mind for this. Compatibility w.r.t. whatever is stored in the database, I mean.

Reply:
Ok, can you check the latest version?